frontend/src/components/dashboard/common/VisitsDensityPeriodDropdown.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+    <DefaultDropdown
+        :items="items"
+        :value="value"
+        @change="changePeriod"
+    />
+</template>
+
+<script>
+    import {period} from "@/services/periodService";
+    import DefaultDropdown from '@/components/common/DefaultDropdown';
+
+    export default {
+        name: "VisitsDensityPeriodDropdown",
+        components: {
+            DefaultDropdown
+        },
+        props: {
+            value: {
+                type:String,
+                required: true
+            },
+            items: {
```

Comment:
Not quite sure if you need to define items as a prop here, it's better to do this in local state, or in computed prop, if you want to make it immutable for sure (not a blocker)